get_function.c

```diff
@@ -0,0 +1,29 @@
+#include "main.h"
+
+/**
+ * find_format_handlers- function is responsible for identifying formats
+ *                        intended for use with _printf.
+ *          It then calls the appropriate corresponding function based on
+ *          the identified format.
+ * @format: The format specifier (char, string, int, decimal, etc)
+ * Return: Either NULL or the associated function pointer.
+ */
+int (*find_format_handlers(const char *format))(va_list arg)
+{
+	unsigned int i;
+
+	i = 0;
+
+	format_handler find_func[] = {
+		{"c", print_char}
```

Comment:
Please update `print_char` to `print_spec_c_match` and add the other function names too
`{"d", print_spec_d_match},
{"i", print_spec_i_match},
{NULL, NULL}`

Reply:
that was an error .. you will do it when working on it